django/publicmapping/redistricting/templates/viewplan.html

```diff
@@ -209,6 +211,15 @@
 
   </head>
   <body onload="init();" id="app">
+    <!-- Load Facebook SDK for JavaScript -->
```

Comment:
Can this Facebook SDK be templated out into another file and just dropped in here instead of copied in both places?

Reply:
Good call! I did this, but in the process noticed that we're not using template inheritance, which makes me 😢